Library/Formula/supercollider.rb

```diff
@@ -0,0 +1,61 @@
+
+class Supercollider < Formula
+  homepage "https://supercollider.github.io"
+  url "https://github.com/supercollider/supercollider.git",
+    :tag => "after-qt5",
+    :revision => "085cab288947fe06f468117f5c294db516b2afc0"
+  sha256 "80956465c2637cf0cc67c3550900e2f392d10d33f2518c25bbb9332901ae226c"
+
+  version "3.7-devel"
+
+  depends_on :xcode => :build
+  depends_on :macos => :lion
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+  depends_on "readline" => :build
+
+  def install
+    ENV.deparallelize
+    bp = buildpath/"build"
+    mkdir_p bp
+    # std_cmake_args +
+    args = [
+      "-G", "Xcode",
+      "-DCMAKE_PREFIX_PATH=#{Formula["qt5"].opt_prefix}",
+      "-DSC_QT=1",
+      # defaults to the user's OS
+      # "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.7",
+      "-DREADLINE_INCLUDE_DIR=#{Formula["readline"].opt_include}",
+      "-DREADLINE_LIBRARY=#{Formula["readline"].opt_lib}/libreadline.dylib",
+      ".."
+    ]
+
+    cd bp do
+      system "cmake", *args
+      xcodebuild "-target", "install",
+                "-configuration", "Release"
```

Comment:
it is being done by cmake, no ?  what specific line or command are you referring to ?

the build should be done in the exact same way as we package releases